scripts/create_baseline_stubs.py

```diff
@@ -0,0 +1,174 @@
+"""Script to generate unannotated baseline stubs using stubgen.
+
+Basic usage:
+$ python scripts/create_baseline_stubs.py <project on PyPI>
+
+Run with -h for more help.
+"""
+
+import argparse
+import os
+import re
+import shutil
+import subprocess
+import sys
+from typing import Optional, Tuple
+
+PYRIGHT_CONFIG = "pyrightconfig.stricter.json"
+
+
+def pip_install(project: str) -> None:
+    print(f"Installing distribution: pip install {project}")
+    subprocess.run(["pip", "install", project], check=True)
```

Comment:
- I don't like how the project remains installed forever, and basically pollutes your venv. Maybe we should check if we are running in a virtualenv? Something like `assert "VIRTUAL_ENV" in os.environ`. This is important, because otherwise `python` or `pip` might use a different Python than the currently running process.
- If the package is already installed, this won't update it. How about `--upgrade`?